### ISSUE TITLE
Sorry, I'm having trouble accessing your connect control for Spotify skill right now

### ISSUE THREAD

**Issue by jagjordi** [thorpelawrence/alexa-spotify-connect#101]
A few days back in started to get this error when trying to use this skill,

> Sorry, I'm having trouble accessing your connect control for Spotify skill right now

I tried removing and adding again the skill but still the same problem.

 Any idea why?

**Comment by thorpelawrence** [comment]
Hey, there's been some problems with Amazon preventing me from migrating away from Heroku, but I haven't had time to fix the issues as I do this in my free time. The issue seems to be related to the translations not always working, and so they're not approving the endpoint change despite being unrelated.

**Comment by jagjordi** [comment]
I see, would there be anything that we could help with? Maybe with the translations?

**Comment by thorpelawrence** [comment]
I did try to fix it a while ago but didn't get far, it seems to be that the translation language can change in the middle of the request (probably when other language requests come in), not sure why it's being set globally instead of per-request

**Comment by jagjordi** [comment]
It might be like that for my case. I have not figured out how to completely set up english as the language, since my amazon account is Spanish, sometimes it gets mixed up, and I get replies from alexa in Spanish even though the question was in english

**Comment by thorpelawrence** [comment]
Yeah, it seems like the Alexa library I'm using isn't working as expected with translations, but it would be too much work to change it
